### PR TITLE
(PA-5326) Added platform for macos13-x86_64

### PR DESCRIPTION
**Pull request by amitkarsale**
Added platform for macos13 intel.

### CODE REVIEW

**Comment by mhashizume** [comment]
Would you be able to rebase on main add this change to the CHANGELOG.md please?

**Comment by mhashizume** [comment]
Hi @amitkarsale , have you gotten a chance to take a look at my review? Thank you

**Comment by amitkarsale** [comment]
> Hi @amitkarsale , have you gotten a chance to take a look at my review? Thank you

Sorry, I was waiting for other changes to be addressed. I have addressed your comment. Thanks @mhashizume !

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ This changelog adheres to [Keep a CHANGELOG](https://keepachangelog.com).
 
 ## [Unreleased]
 
+## Added
+-(PA-5326) Added macos 13 (x86_64)
+
 ### Added
```
Comment (mhashizume) [inline CHANGELOG.md:12]: Can you append your change to the "Added" list that has already been started instead of creating a new "Added" list? Thanks

**Review by mhashizume — APPROVED** [review]

**Review by e-gris — APPROVED** [review]

**Pull request merged**